extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -974,16 +980,11 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
    * Static variables and methods.
    */
   companion object {
-    private val workerThread by lazy {
-      HandlerThread("STYLE_WORKER").apply {
-        priority = Thread.MAX_PRIORITY
-        start()
-      }
-    }
-    private val workerHandler by lazy {
-      Handler(workerThread.looper)
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    var workerThread = HandlerThread("STYLE_WORKER").apply {
```

Comment:
Could we revert lazy initialization? That thread in some situations may not even be started (if user does not use geojson source). I'm OK to leave it internal for tests.

Reply:
This is a compromise for tests. With lazy initialization, only the first case will succeed, in the following one, this thread will be in the inactive state.